homeassistant/components/niko_home_control/const.py

```diff
@@ -4,3 +4,10 @@
 
 DOMAIN = "niko_home_control"
 _LOGGER = logging.getLogger(__name__)
+
+
+NIKO_THERMOSTAT_MODES_MAP = {
+    "off": 3,
```

Comment:
capitalize constants

```suggestion
    "OFF": 3,
```